src/renderer/pages/Project.tsx

```diff
@@ -29,6 +31,9 @@ const ProjectPage = () => {
   const { isExporting, exportProgress } = useSelector(
     (store: ApplicationStore) => store.exportIo
   );
+  const openUserFeedback = () => {
+    ipc.openExternalLink('https://forms.gle/YNozsPZxF4kxSuud8');
```

Comment:
I reckon we should put all URL constants like this in one place - make a constants file and put this as well as the other URLs (e.g. the link to AssemblyAI signup) in there

Reply:
Yep, great idea. Do you want to create a file in this PR?

Reply:
Done